help/md/dnBimodalLognormal.md

```diff
@@ -3,8 +3,9 @@ dnBimodalLognormal
 ## title
 Bimodal Lognormal Distribution
 ## description
-A bimodal lognormal distribution, that is, with probability p a value is distributed according to the first lognormal distribution and with probability 1-p from the second lognormal distribution.
+The Bimodal Lognormal distribution unites two separate lognormal distributions.
 ## details
+The bimodal lognormal distribution takes on five parameters: mean1 and mean2 (the means of two lognormal distributions), sd1 and sd2 (the standard deviations of two lognormal distributions), and p (between 0 and 1). The value will be distributed according to the first lognormal distribution with probability p, and distributed according to the second lognormal distribution with probability (1 - p).
```

Comment:
Are the means and standard deviations on the log scale?  The LogNormal distribution is the exponential of a normal distribution, so if we have

X = exp(Normal(mu,sigma))

then `mean(X) != mu`.  The mu is the mean on the log scale, and `mean(X)` is different.